lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -72,44 +72,50 @@ function dosomething_northstar_verify_user($credentials) {
 }
 
 /**
+ * Save the user's Northstar ID to their local profile.
+ *
+ * @param $user - Drupal user
+ * @param $northstar_response - Northstar user JSON response
+ */
+function dosomething_northstar_save_id_field($user, $northstar_response) {
```

Comment:
we have some [getters](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1171) and [setters](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1235) in the user module, maybe this should live there?